client/js/components/assessment_results/result_summary.jsx

```diff
@@ -1,25 +1,12 @@
 "use strict";
 
 import React                 from "react";
-import { connect }           from "react-redux";
 import _                     from "lodash";
 
 import AssessmentStore       from "../../stores/assessment";
 import ReviewAssessmentStore from "../../stores/review_assessment";
 import ItemResult            from "./item_result";
 
-import { outcomes }          from "../../selectors/assessment";
-
-const select = (state, props) => {
-  return {
-    assessmentResult : AssessmentStore.assessmentResult(), // TODO this won't work. have to figure out where to get assessmentResult from
-    outcomes         : outcomes(),
-    settings         : state.settings,
```

Comment:
Where did the assessmentResult go? Won't we need selectors to get data from state for this component?